site/docs/concepts/optimization-techniques/kvcache-eviction-algorithm.md

```diff
@@ -3,3 +3,55 @@ sidebar_position: 2
 ---
 
 # KVCache Token Eviction Algorithm
+
+
+## Overview
+The cache eviction algorithm is designed to manage KV (Key-Value) cache memory for large language models (LLMs) during text generation. It determines which blocks of tokens should be evicted from the KV cache based on importance scores calculated from attention scores across different attention layers.
+
+## Conceptual Model
+The KV cache for each sequence is divided into three logical areas:
+
+![KV cache layout with cache eviction](./images/kv-cache-areas-diagram.svg)
```

Comment:
Images should be placed at `site/static/img` directory and referenced here from this directory:
```suggestion
![KV cache layout with cache eviction](/img/kv-cache-areas-diagram.svg)
```

Reply:
Done